From 46dc8df9c85b74ab0b90c994d8e221b761e92ceb Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Tue, 6 Mar 2018 10:23:23 +0530
Subject: [PATCH 0316/1851] dt: bindings: media: xilinx: Add YUV420 decoding
 support in UHDSDI Rx

commit fe26e07c157d743d7c8140d008811ad9ec6fe94e from
https://github.com/Xilinx/linux-xlnx.git

Updates the documentation for supporting YUV420 10bpc
in UHDSDI Rx Subsystem driver.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,sdirxss.txt       | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
index e5c46dc6187c..7da5c5d70bee 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
@@ -7,7 +7,7 @@ It outputs the video as an AXI4 Stream video data in YUV 422 10bpc mode.
 The subsystem consists of the SDI Rx IP whose SDI native output is connected
 to a SDI to Native conversion Bridge. The output of the Native bridge is
 connected to a Native to AXI4S Bridge which generates the AXI4 Stream of
-YUV422 10 bpc in dual pixel per clock. This output is fixed.
+YUV422 or YUV420 10 bpc in dual pixel per clock.
 
 Required properties:
 
@@ -27,8 +27,8 @@ Required properties:
   The SDI Rx subsystem has one port configured as output port.
 
 - xlnx,video-format, xlnx,video-width: Video format and width, as defined in
-  video.txt. Please note that the video format is fixed to YUV422 and the
-  video-width is 10.
+  video.txt. Please note that the video format is fixed to either YUV422 or YUV420
+  and the video-width is 10.
 
 Example:
 		v_smpte_uhdsdi_rx_ss: v_smpte_uhdsdi_rx_ss@80000000 {
-- 
2.31.1

